client-src/elements/chromedash-metadata.ts

```diff
@@ -1,39 +1,33 @@
 import {LitElement, css, html} from 'lit';
+import {customElement, property, state} from 'lit/decorators.js';
 import {ifDefined} from 'lit/directives/if-defined.js';
 import {SHARED_STYLES} from '../css/shared-css.js';
 
+@customElement('chromedash-metadata')
 class ChromedashMetadata extends LitElement {
-  static get properties() {
-    return {
-      implStatuses: {type: Array}, // Read in chromedash-featurelist.
-      status: {attribute: false}, // Read in chromedash-featurelist.
-      selected: {attribute: false}, // Directly edited in /templates/features.html
-      _className: {attribute: false},
-      _fetchError: {attribute: false},
-      _channels: {attribute: false},
-      _versions: {attribute: false},
-    };
-  }
-
-  constructor() {
-    super();
-    this.implStatuses = [];
-    this.status = {
-      NO_ACTIVE_DEV: 1,
-      PROPOSED: 2,
-      IN_DEVELOPMENT: 3,
-      BEHIND_A_FLAG: 4,
-      ENABLED_BY_DEFAULT: 5,
-      DEPRECATED: 6,
-      REMOVED: 7,
-      ORIGINTRIAL: 8,
-      INTERVENTION: 9,
-      ON_HOLD: 10,
-      NO_LONGER_PURSUING: 1000,
-    };
-    this._channels = {};
-    this._versions = [];
-  }
+  @property({type: Array})
+  implStatuses;
+  @property({type: Object, attribute: false})
+  status = {
+    NO_ACTIVE_DEV: 1,
+    PROPOSED: 2,
+    IN_DEVELOPMENT: 3,
+    BEHIND_A_FLAG: 4,
+    ENABLED_BY_DEFAULT: 5,
+    DEPRECATED: 6,
+    REMOVED: 7,
+    ORIGINTRIAL: 8,
+    INTERVENTION: 9,
+    ON_HOLD: 10,
+    NO_LONGER_PURSUING: 1000,
+  };
+
+  @state()
+  selected;
```

Comment:
This is directly modified at https://github.com/GoogleChrome/chromium-dashboard/blob/c975330f24e750a4adc96de955a50be8f8c11a23/client-src/js-src/features-page.js#L54 so it needs to stay a `@property({attribute:false})`.